ecommerce_integrations/woocommerce/doctype/woocommerce_setting/test_woocommerce_setting.py

```diff
@@ -0,0 +1,32 @@
+# Copyright (c) 2021, Frappe and Contributors
+# See license.txt
+
+import unittest
+
+import frappe
+
+from ecommerce_integrations.woocommerce.constants import MODULE_NAME, SETTINGS_DOCTYPE
+
+from .woocommerce_setting import WoocommerceSetting, generate_secret
```

Comment:
unused import... also avoid using relative imports 😅

Reply:
Will remove unused import and fix path.